### PR TITLE
IGAPP-564: Send signal on feedback

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/src/routes/feedback/containers/FeedbackModalContainer.js [hunk 1]
```diff
@@ -1,81 +1,21 @@
...
-      cities={props.cities}
-    />
+    <FeedbackContainer routeType={routeType} feedbackOrigin={feedbackOrigin} language={language} cityCode={cityCode} />
   )
 }
 
-export default connect<PropsType, OwnPropsType, _, _, _, _>(
-  mapStateToProps,
-  mapDispatchToProps
-)(withPayloadProvider<InnerPropsType, OwnPropsType, FeedbackModalRouteType>(refresh)(FeedbackModalContainer))
```
Comment (Taggotty) [inline native/src/routes/feedback/containers/FeedbackModalContainer.js:81]: Am I correct here? Without cities, we don't need `connect` and `wihtPayloadProvider`, right?

Comment (SteffiStuffel) [reply native/src/routes/feedback/containers/FeedbackModalContainer.js:81]: yes thats right. We had a own ticket for it IGAPP-612

**Review by SteffiStuffel — APPROVED** [review]
Tested on Android.

**Review by klinzo — APPROVED** [review]
Tested on Android :+1:

**Comment by SteffiStuffel** [comment]
Not sure if it is supposed to be like this, but no information is send where the feedback was send (offers, categories, search.. ) but we send signals for every page opened so it should be oke, i guess?

**Pull request merged**